flow/include/flow/flow.h

```diff
@@ -1079,7 +1192,9 @@ class Promise final {
 };
 
 template <class T>
-struct NotifiedQueue : private SingleCallback<T>, FastAllocated<NotifiedQueue<T>> {
+struct NotifiedQueue : private SingleCallback<T>
+//    , FastAllocated<NotifiedQueue<T>> // FIXME(swift): Swift can't deal with this type yet
```

Comment:
This changes how `FastAllocated<NotifiedQueue<T>>` is allocated and may affect performance. Can you enable it for C++ build?